cmd/fossa/cmd/report/licenses.go

```diff
@@ -0,0 +1,90 @@
+package report
+
+import (
+	"fmt"
+	"text/template"
+
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/api/fossa"
+	"github.com/fossas/fossa-cli/cmd/fossa/flags"
+	"github.com/fossas/fossa-cli/log"
+)
+
+const defaultLicenceReportTemplate = `# 3rd-Party Software License Notice
+Generated by fossa-cli (https://github.com/fossas/fossa-cli).
+This software includes the following software and licenses:
+{{range $license, $deps := .}}
+========================================================================
+{{$license}}
+========================================================================
+The following software have components provided under the terms of this license:
+{{range $i, $dep := $deps}}
+- {{$dep.Project.Title}} (from {{$dep.Project.URL}})
+{{- end}}
+{{end}}
+`
+
+var licensesCmd = cli.Command{
+	Name:  "licenses",
+	Usage: "Generate licenses report",
+	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(flags.WithModulesFlags([]cli.Flag{
+		cli.StringFlag{Name: flags.Short(Output), Destination: &outputFlag, Value: "-", Usage: "Output file for report"},
+		cli.StringFlag{Name: flags.Short(Template), Destination: &templateFlag, Usage: "process report via template prior to sending it to output"},
+		cli.BoolFlag{Name: flags.Short(Unknown), Usage: "license report including unkown (warning this is SLOW)"},
+	}))),
+	Before: prepareReportCtx,
+	Action: generateLicenses,
+}
+
+func generateLicenses(ctx *cli.Context) (err error) {
+	defer log.StopSpinner()
+	revs := make(fossa.Revisions, 0)
+	for _, module := range analyzed {
+		if ctx.Bool(Unknown) {
+			totalDeps := len(module.Deps)
+			i := 0
+			for _, dep := range module.Deps {
+				i++
+				log.ShowSpinner(fmt.Sprintf("Fetching Licence Info (%d/%d): %s", i+1, totalDeps, dep.ID.Name))
```

Comment:
[nit] In general, prefer American English common usage ("license") for the sake of uniformity.